src/pages/net-neutrality/call-page.js

```diff
@@ -0,0 +1,54 @@
+import React  from 'react';
+import Footer from '../../components/net-neutrality/footer.js';
+import Header from '../../components/net-neutrality/header.js';
+import { SimpleNav } from '../../components/net-neutrality/nav.js';
+import CallButton from '../../components/net-neutrality/call-button.js';
+import SignupForm from '../../components/net-neutrality/signup-form-call.js';
+
+var Impact = React.createClass({
+  render: function() {
+    return (
+      <div className="impact-item-container">
+        <img height="219" width="237" src={this.props.icon}/>
+        <div>
+          <h2>{this.props.header}</h2>
+          <p>{this.props.children}</p>
+        </div>
+      </div>
+    );
+  }
+});
+
+var CallPage = React.createClass({
+  contextTypes: {
+    intl: React.PropTypes.object
+  },
+  render: function() {
+    var className = "home";
+    if (this.props.test) {
+      className += " " + this.props.test;
+    }
+    return (
+      <div className={className}>
+        <div className="page">
+          <div id="home" className="nav-anchor nav-offset"></div>
+          <SimpleNav active="home"/>
+          <Header/>
+          <section>
+            <h1>
+              How will this impact me?
+            </h1>
+            <p>Last week Federal Communications Commission Chairman Ajit Pai revealed the anti-net-neutrality proposal his agency is set to vote on in 13 days. <b>These rules are way, way, WAY worse than expected</b>.</p>
```

Comment:
Vote `on December 14` or make this logic count to December 14

Reply:
Lol, I didn't even read the copy.